addons/mail/static/src/owl/components/chatter/chatter.scss

```diff
@@ -29,3 +29,7 @@
     border-left: 1px solid gray('300');
     background-color: white;
 }
+
+.o_ChatterTopbar {
+    background: none;
```

Comment:
Maybe add a comment in the code because I don't understand why we need that rule.

Reply:
He's not even in the right file, I move it in the right file and add a comment :)